app/models/card.rb

```diff
@@ -1,9 +1,14 @@
 class Card < ActiveRecord::Base
-  before_validation :set_review_date, on: :create
+  before_validation :set_review_date, on: [:create]
   validate :original_translated_text_equal
   validates :original_text, :translated_text, :review_date,
             presence: { message: 'Необходимо заполнить поле.' }
 
+  scope :review_card, -> { where('review_date <= ?', Time.now.strftime('%Y-%m-%d')).order('RANDOM()') }
```

Comment:
Line is too long. [103/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.